tests/data_context/test_data_context_test_yaml_config.py

```diff
@@ -22,45 +22,6 @@
 from tests.test_utils import create_files_in_directory, set_directory
 
 
-@pytest.fixture
```

Comment:
@NathanFarmer How come this fixture is being deleted?  Is it being moved elsewhere?

Reply:
@alexsherstinsky Yes, this was moved to back to `tests/conftest.py`. Somebody moved it in the middle of this PR, but I want to use it too 😄

Reply:
See above to see it added in `tests/conftest.py`